assets/js/components/heading.tsx

```diff
@@ -15,11 +16,12 @@ export function Heading({
   level,
   children,
   id = urlify(children),
+  className,
 }: HeadingProps) {
```

Comment:
Could also consider forwarding along any valid HTML props.

```suggestion
  ...props,
}: HeadingProps & JSX.HTMLAttributes<HTMLHeadingElement>) {
```

Reply:
Ok! And we'd want to remove those from the definition of HeadingProps then, right?

Reply:
Yeah there's a few more revisions that would need to be done in addition to that suggestion (also importing the `JSX` type and applying the props to `TagName`). The only thing we'd need to remove from the prop type is `className`.

Reply:
y'know, in retrospect, I think this is fine as-is, if you want to just leave it.

Reply:
LOL ok will leave it